packages/ag-charts-community/src/module/moduleMap.ts

```diff
@@ -55,4 +55,10 @@ export class ModuleMap<M extends Module<C, I>, C, I extends ModuleInstance = Mod
     isModuleEnabled(module: M) {
         return this.modules[module.optionsKey] != null;
     }
+
+    *[Symbol.iterator](): IterableIterator<I> {
+        for (const { instance } of Object.values(this.modules)) {
+            yield instance;
```

Comment:
Interesting, this is the first practical use of `yield` I've seen in a codebase :D

Reply:
I actually think that it might make more sense for `Series` to extend `ModuleMap` and have `this.modules` be protected, but that's a discussion for another time 😄